src/form-elements/FormElementFiles/FormElementFileDisplay.tsx

```diff
@@ -1,26 +1,32 @@
 import * as React from 'react'
 import { AttachmentValid } from '../../hooks/attachments/useAttachments'
-
+import OnLoading from '../../components/OnLoading'
+import { Tooltip } from '@material-ui/core'
 interface Props {
-  file: AttachmentValid
+  attachment: AttachmentValid
+  attachmentBlob: Blob | undefined
+  isLoadingAttachmentBlob: boolean
 }
 
-const FormElementFileDisplay = ({ file }: Props) => {
+const FormElementFileDisplay = ({
+  attachment,
+  attachmentBlob,
+  isLoadingAttachmentBlob,
+}: Props) => {
   const isImageType = React.useMemo(() => {
-    if (!file.type) {
-      return file.contentType.includes('image/')
+    if (!attachment.type) {
+      return attachment.contentType.includes('image/')
     }
-    return file.data.type.includes('image/')
-  }, [file])
+    return attachment.data.type.includes('image/')
+  }, [attachment])
   const imageUrl = React.useMemo(() => {
-    if (file.type === 'NEW' || file.type === 'SAVING') {
-      return URL.createObjectURL(file.data)
+    if (attachment.type === 'NEW' || attachment.type === 'SAVING') {
+      return URL.createObjectURL(attachment.data)
     }
-    if (!file.type) {
-      // TODO: Load image
-      return
+    if (!attachment.type && attachmentBlob) {
+      return URL.createObjectURL(attachmentBlob)
```

Comment:
These need to be cleaned up with https://developer.mozilla.org/en-US/docs/Web/API/URL/revokeObjectURL at some point